### PR TITLE
feature/add_option_to_show_exerpt_article_link

### PR DESCRIPTION
**Pull request by border0464111**
This feature add a checkbox on the settings page to allow the display of post exerpt under pagelink for archive template.

### CODE REVIEW

**Comment by ajaydsouza** [comment]
It's a problem with the tests on 7.0 that I need to fix. It used to run fine earlier

File: includes/main.php [hunk 1]
```diff
@@ -260,12 +260,17 @@ function wzkb_article_header( $term, $level ) {
...
+	$wzkb_options = get_option('wzkb_settings',false);
+
 	$output = '<ul class="wzkb-articles-list term-' . $term->term_id . '">';
 
 	while ( $query->have_posts() ) : $query->the_post();
 
 		$output .= '<li class="wzkb-article-name post-' . get_the_ID() . '">';
 		$output .= '<a href="' . get_permalink( get_the_ID() ) . '" rel="bookmark" title="' . get_the_title( get_the_ID() ) . '">' . get_the_title( get_the_ID() ) . '</a>';
+		if ( $wzkb_options['show_exerpt'] ) {
+			$output .= '<div class="wzkb-article-exerpt post-' . get_the_ID() . '" >'.get_the_excerpt( get_the_ID() ) . '</div>';
```
Comment (ajaydsouza) [inline includes/main.php]: Please format based on WPCS, i.e. space before and after the `.`

Spelling of `excerpt`

File: includes/main.php [hunk 2]
```diff
@@ -260,12 +260,17 @@ function wzkb_article_header( $term, $level ) {
  */
 function wzkb_article_loop( $term, $level, $query ) {
 
+	$wzkb_options = get_option('wzkb_settings',false);
```
Comment (ajaydsouza) [inline includes/main.php]: Use `wzkb_get_option`

File: includes/main.php [hunk 3]
```diff
@@ -260,12 +260,17 @@ function wzkb_article_header( $term, $level ) {
...
+
 	$output = '<ul class="wzkb-articles-list term-' . $term->term_id . '">';
 
 	while ( $query->have_posts() ) : $query->the_post();
 
 		$output .= '<li class="wzkb-article-name post-' . get_the_ID() . '">';
 		$output .= '<a href="' . get_permalink( get_the_ID() ) . '" rel="bookmark" title="' . get_the_title( get_the_ID() ) . '">' . get_the_title( get_the_ID() ) . '</a>';
+		if ( $wzkb_options['show_exerpt'] ) {
```
Comment (ajaydsouza) [inline includes/main.php]: See line 263

File: includes/admin/register-settings.php [hunk 4]
```diff
@@ -234,6 +234,13 @@ function wzkb_get_registered_settings() {
 					'type'             => 'text',
 					'options'          => 'kb-tags',
 				),
+				'show_exerpt'     => array(
```
Comment (ajaydsouza) [inline includes/admin/register-settings.php]: Fix spelling of `excerpt` throughout the request

File: includes/admin/register-settings.php [hunk 5]
```diff
@@ -234,6 +234,13 @@ function wzkb_get_registered_settings() {
...
+					'id'               => 'show_exerpt',
+					'name'             => esc_html__( 'Show exerpt', 'knowledgebase' ),
+					'desc'             => esc_html__( 'Show exerpt under article link', 'knowledgebase' ),
```
Comment (ajaydsouza) [inline includes/admin/register-settings.php]: Description should be "Select to include the post excerpt after the article link"

**Comment by border0464111** [comment]
Oh, that's unfortunate. Do you need me to pull current develop state and create a new pull request  
Or you can resolve it live. ?

**Comment by ajaydsouza** [comment]
That would be ideal. I noticed some merge issues that need to be manually resolved with the other pr

**Comment by border0464111** [comment]
I solved the conflicts, merging should be good now

**Comment by ajaydsouza** [comment]
The travis seems to be failing with an error on Line 277. Could you please check that to see if that is fine?

**Pull request merged**